specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/WafPolicyCreateOrUpdate.json

```diff
@@ -111,6 +111,59 @@
               "matchVariable": "RequestArgValues",
               "selectorMatchOperator": "StartsWith",
               "selector": "test"
+            },
+            {
+              "exclude": "Request",
```

Comment:
Is there any example of "exclude": "MatchVariable"?

Reply:
Will add one with matchVariable